jest/jest-setup-after-env.ts

```diff
@@ -40,7 +39,7 @@ expect.addSnapshotSerializer(replaceAttributesSnapshotSerializer([
   'data-tableid',
 ]))
 
-const server = setupMockServer(setupHandlers(import.meta.env.VITE_KUMA_API_SERVER_URL, mocks))
+const server: SetupServer = setupServer(...get(TOKENS.mswHandlers))
```

Comment:
**Suggestion**: This is already correctly inferred and shouldn’t have a type annotation.

```suggestion
const server = setupServer(...get(TOKENS.mswHandlers))
```

Reply:
Done!